tests/integration/test_benchmark.py

```diff
@@ -13,3 +13,23 @@ def test_identity():
     scores = output.groupby('synthesizer').score.mean().sort_values()
 
     assert ['Uniform', 'Independent', 'Identity'] == scores.index.tolist()
+
+
+def test_identity_jobs():
```

Comment:
Can we also add a test to validate the passing of a single synthesizer as a serialized JSON?